virttest/qemu_vm.py

```diff
@@ -478,16 +478,26 @@ def add_qmp_monitor(devices, monitor_name, filename):
                              " unsupported")
                 return add_human_monitor(devices, monitor_name, filename)
 
+            chardev_params = params.object_params(monitor_name)
+            backend = chardev_params.get('chardev_backend', 'unix_socket')
+            host = chardev_params.get('chardev_host', '127.0.0.1')
+            port = str(utils_misc.find_free_ports(5000, 6000, 1, host)[0])
```

Comment:
Check if it support chardev first, then set params.

Reply:
Thanks and update.